src/main/scala/thurloe/service/DataAccess.scala

```diff
@@ -0,0 +1,12 @@
+package thurloe.service
+
+import scala.util.Try
+
+trait DataAccess {
+  def setKeyValuePair(keyValuePair: KeyValuePair): Try[KeyValuePair]
```

Comment:
Yes I think this should be `Try[Unit]`, with a successful status code back to the client when this is a `Success`.